File: java/bypstest-api/src/byps/test/api/remote/RemoteEnums.java
package byps.test.api.remote;

import byps.BRemote;
import byps.RemoteException;
import byps.test.api.enu.EnumPlanets;

/**
 * 
 */
public interface RemoteEnums extends BRemote {

	EnumPlanets getPlanet() throws RemoteException;
	
	void setPlanet(EnumPlanets planet) throws RemoteException;
	
}

File: csharp/bypstest-ser/src-ser/byps/test/api/remote/RemoteEnums.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.remote
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface RemoteEnums : BRemote {
	
		byps.test.api.enu.EnumPlanets GetPlanet();
		void GetPlanet(BAsyncResult<byps.test.api.enu.EnumPlanets> asyncResult) ;
		Task<byps.test.api.enu.EnumPlanets> GetPlanetAsync();
		
		void SetPlanet(byps.test.api.enu.EnumPlanets planet);
		void SetPlanet(byps.test.api.enu.EnumPlanets planet, BAsyncResult<Object> asyncResult) ;
		Task SetPlanetAsync(byps.test.api.enu.EnumPlanets planet);
		
		
	}
}  // end namespace
